src/analysis/classic/hunter/modules/features/AlwaysBeCasting.tsx

```diff
@@ -0,0 +1,83 @@
+import { t } from '@lingui/macro';
+import { formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS/classic/hunter';
+import { SpellLink } from 'interface';
+import { ThresholdStyle, When } from 'parser/core/ParseResults';
+import CoreAlwaysBeCasting from 'parser/shared/modules/AlwaysBeCasting';
+import Gauge from 'parser/ui/Gauge';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
+  position = STATISTIC_ORDER.CORE(6);
+
+  get downtimeSuggestionThresholds() {
+    return {
+      actual: this.downtimePercentage,
+      isGreaterThan: {
+        minor: 0.1,
+        average: 0.15,
+        major: 0.2,
+      },
+      style: ThresholdStyle.PERCENTAGE,
+    };
+  }
+
+  suggestions(when: When) {
+    when(this.downtimeSuggestionThresholds)
+      .isGreaterThan(0.15)
+      .addSuggestion((suggest, actual, recommended) =>
+        suggest(
+          <>
+            Your downtime can be improved. Try to Always Be Casting (ABC). If you have to move, use
+            instant cast spells, such as
+            {/* UPDATE THE SPELLS BELOW */}
```

Comment:
Remove update comments once you have updated.

Reply:
Done , I think I got them all.